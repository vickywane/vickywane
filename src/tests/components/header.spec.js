/**
 * @jest-environment jsdom
 */
import React from "react"
import renderer from "react-test-renderer"
import { Router } from "@reach/router"

import "@testing-library/jest-dom/extend-expect"
import Header from "../../components/header"

describe("Header component test", () => {
  it("It displays headers", function () {
    const tree = renderer
      .create(
        <Router>
          <Header path="/" exact />
        </Router>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
