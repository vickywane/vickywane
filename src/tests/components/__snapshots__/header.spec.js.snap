// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test It displays headers 1`] = `
<div
  style={
    Object {
      "outline": "none",
    }
  }
  tabIndex="-1"
>
  <header
    className="sc-iqAbSa dgIjuy"
    data-testid="header-component"
  >
    <span>
      <a
        href="/"
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "justifyContent": "center",
            }
          }
        >
          <img
            alt="me"
            className="sc-crzoUp fRCiKF"
            src="test-file-stub"
          />
        </div>
      </a>
      <ul
        className="sc-dIsAE dbBDIb"
      >
        <li>
          <div
            className="sc-hKFyIo fLlqAt"
            color="white"
          >
            <a
              href="/blog"
            >
              <p
                className="sc-eCApGN kwJQEo"
              >
                 
                Articles
                 
              </p>
            </a>
          </div>
        </li>
        <li>
          <div
            className="sc-hKFyIo fLlqAt"
            color="white"
          >
            <a
              href="/home/experiences"
            >
              <p
                className="sc-eCApGN kwJQEo"
              >
                 
                Work Experience
                 
              </p>
            </a>
          </div>
        </li>
        <li>
          <div
            className="sc-hKFyIo fLlqAt"
            color="white"
          >
            <a
              href="/projects"
            >
              <p
                className="sc-eCApGN kwJQEo"
              >
                 
                Projects
                 
              </p>
            </a>
          </div>
        </li>
        <li>
          <div
            className="sc-hKFyIo fLlqAt"
            color="white"
          >
            <a
              href="/talks"
            >
              <p
                className="sc-eCApGN kwJQEo"
              >
                 
                Talks
                 
              </p>
            </a>
          </div>
        </li>
      </ul>
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
          }
        }
      >
        <div>
          <div
            className="sc-jrsJCI dLfXIF"
          >
            <div
              className="sc-fujyUd HVVod"
              onClick={[Function]}
            >
              <svg
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  Object {
                    "color": undefined,
                    "fontSize": "1.8rem",
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <line
                  x1="3"
                  x2="21"
                  y1="12"
                  y2="12"
                />
                <line
                  x1="3"
                  x2="21"
                  y1="6"
                  y2="6"
                />
                <line
                  x1="3"
                  x2="21"
                  y1="18"
                  y2="18"
                />
              </svg>
            </div>
          </div>
          <nav
            className="sc-pNWxx eJKgmt"
            open={false}
          >
            <ul
              className="sc-kEqYlL gholCc"
            >
              <li>
                <div
                  className="sc-hKFyIo fLlqAt"
                  color="white"
                >
                  <a
                    href="/blog"
                  >
                    <p
                      className="sc-eCApGN kwJQEo"
                    >
                       
                      Articles
                       
                    </p>
                  </a>
                </div>
                <hr
                  style={
                    Object {
                      "background": "white",
                      "margin": ".9rem 0",
                    }
                  }
                />
              </li>
              <li>
                <div
                  className="sc-hKFyIo fLlqAt"
                  color="white"
                >
                  <a
                    href="/home/experiences"
                  >
                    <p
                      className="sc-eCApGN kwJQEo"
                    >
                       
                      Work Experience
                       
                    </p>
                  </a>
                </div>
                <hr
                  style={
                    Object {
                      "background": "white",
                      "margin": ".9rem 0",
                    }
                  }
                />
              </li>
              <li>
                <div
                  className="sc-hKFyIo fLlqAt"
                  color="white"
                >
                  <a
                    href="/projects"
                  >
                    <p
                      className="sc-eCApGN kwJQEo"
                    >
                       
                      Projects
                       
                    </p>
                  </a>
                </div>
                <hr
                  style={
                    Object {
                      "background": "white",
                      "margin": ".9rem 0",
                    }
                  }
                />
              </li>
              <li>
                <div
                  className="sc-hKFyIo fLlqAt"
                  color="white"
                >
                  <a
                    href="/talks"
                  >
                    <p
                      className="sc-eCApGN kwJQEo"
                    >
                       
                      Talks
                       
                    </p>
                  </a>
                </div>
                <hr
                  style={
                    Object {
                      "background": "white",
                      "margin": ".9rem 0",
                    }
                  }
                />
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </span>
  </header>
</div>
`;
